samples/cpp/whisper_speech_recognition/audio_utils.cpp

```diff
@@ -87,7 +87,7 @@ ov::genai::RawSpeechInput read_wav(const std::string& filename) {
     }
 
     const uint64_t n =
-        wav_data.empty() ? wav.totalPCMFrameCount : wav_data.size() / (wav.channels * wav.bitsPerSample / 8);
+        wav_data.empty() ? wav.totalPCMFrameCount : wav_data.size() / (static_cast<uint64_t>(wav.channels) * wav.bitsPerSample / 8);
```

Comment:
Do we need cast to uint64_t for `wav.bitsPerSample`? 
It has `drwav_uint16` type as well.

Reply:
the point is `/ 8`. 8 is a int literal. So the whole expression is converted to int. 
Changed 8 literal type to unsigned long.